src/Command/Pull/PullCommandBase.php

```diff
@@ -244,6 +249,11 @@ protected function downloadDatabaseDump(
   ): string {
     $database_backups = new DatabaseBackups($acquia_cloud_client);
     $backups_response = $database_backups->getAll($environment->uuid, $database->name);
+    if (!count($backups_response)) {
+      $this->logger->warning('No existing backups found, creating an on-demand backup now. This will take some time depending on the size of the database.');
```

Comment:
```suggestion
      $this->logger->warning('No existing database backups found, creating a new backup now. This will take some time depending on the size of the database.');
```